editor/inserter/index.js

```diff
@@ -88,13 +94,18 @@ class Inserter extends wp.element.Component {
 }
 
 export default connect(
-	undefined,
+	( state ) => {
+		return {
+			selectedBlock: getSelectedBlock( state ),
```

Comment:
Are you suggesting we trigger the `insertBlock` action here without specifying the position where we'll insert the block (see `after` argument in `onInsertBlock`)?

I guess this is possible, but this complexifies the `blocks.order` reducer a lot, because we'll need to be aware of the `insertedBlock` sub-reducer value inside the `blocks.order` reducer to find the right position. Which means we can't use `combineReducers` utility.

I think we shouldn't introduce this complexity in the state and instead have all the required information to process the insert in the INSERT action.